datasync/chaindatafetcher/api.go

```diff
@@ -39,8 +39,28 @@ func (api *PublicChainDataFetcherAPI) StopFetching() error {
 	return api.f.stopFetching()
 }
 
-func (api *PublicChainDataFetcherAPI) StartRangeFetching(start, end uint64, reqType uint) error {
-	return api.f.startRangeFetching(start, end, types.RequestType(reqType))
+func (api *PublicChainDataFetcherAPI) StartRangeFetching(start, end uint64, reqType interface{}) error {
```

Comment:
@jeongkyun-oh Can we make test cases for this API?

Reply:
@kjhman21 I feel creating unittest for this is too much. I have tested locally. Any concern for this?

Reply:
@jeongkyun-oh Okay. got it!